homeassistant/components/wiz/light.py

```diff
@@ -0,0 +1,348 @@
+"""WiZ integration."""
+from __future__ import annotations
+
+from datetime import timedelta
+import logging
+
+from pywizlight import PilotBuilder, wizlight
+from pywizlight.bulblibrary import BulbClass, BulbType
+from pywizlight.exceptions import WizLightNotKnownBulb, WizLightTimeOutError
+from pywizlight.rgbcw import convertHSfromRGBCW
+from pywizlight.scenes import get_id_from_scene_name
+
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS,
+    ATTR_COLOR_TEMP,
+    ATTR_EFFECT,
+    ATTR_HS_COLOR,
+    ATTR_RGB_COLOR,
+    SUPPORT_BRIGHTNESS,
+    SUPPORT_COLOR,
+    SUPPORT_COLOR_TEMP,
+    SUPPORT_EFFECT,
+    LightEntity,
+)
+from homeassistant.const import CONF_NAME
+from homeassistant.helpers.device_registry import CONNECTION_NETWORK_MAC
+import homeassistant.util.color as color_utils
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+SUPPORT_FEATURES_RGB = (
+    SUPPORT_BRIGHTNESS | SUPPORT_COLOR | SUPPORT_COLOR_TEMP | SUPPORT_EFFECT
```

Comment:
We're using deprecated supported features here. We should use the light color mode feature instead.

https://developers.home-assistant.io/docs/core/entity/light#color-modes

Reply:
#65746